src/wp-includes/cache-compat.php

```diff
@@ -35,3 +35,21 @@ function wp_cache_get_multiple( $keys, $group = '', $force = false ) {
 		return $values;
 	}
 endif;
+
+if ( ! function_exists( 'wp_cache_flush_runtime' ) ) :
+	/**
+	 * Removes all cache items, from the in-memory runtime cache.
+	 *
+	 * Compat function to mimic wp_cache_flush_runtime().
+	 *
+	 * @ignore
+	 * @since 6.0.0
+	 *
+	 * @see wp_cache_flush_runtime()
+	 *
+	 * @return bool False on failure, true on success
+	 */
+	function wp_cache_flush_runtime() {
```

Comment:
would it make sense to add a test specifically for this function?

Reply:
@adamsilverstein: I can't find tests for the `wp_cache_get_multiple()` compat function above.

I'm not sure how to test this, since it will already be defined in `src/wp-includes/cache.php`.

Reply:
This function doesn't really do anything in this implementation. It is only not have a test here, as a test wouldn't add any value.